CMakePresets.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 16,
```

Comment:
`CMakePresets.json` is available since CMake 3.19: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
So 3.16 is meaningless.
And `"version": 3` is available since CMake 3.21.

Reply:
Thank you so much. I misunderstood we should use `"version": 1` and specify `CMake 3.19` here.

Reply:
fix: [23a53c5](https://github.com/mroonga/mroonga/pull/705/commits/23a53c5e13f89af8026d72d17eef2fe61aab205f)